python/lsst/sims/catUtils/mixins/sncat.py

```diff
@@ -1,23 +1,33 @@
 """
-Mixin to InstanceCatalog class to give SN catalogs in catsim
+Mixins for the InstanceCatalog class to provide SN catalogs in catsim. There
+three classes here:
+    - SNFunctionality: provides common functions required by all SN instance
+        catalogs. It does not make sense to instantiate this class, but rather
+        it should be used as a mixin alongside another class.
+    - SNIaCatalog: Dynamically created catalogs by sampling user specified
+        distributions of SN parameters on the fly based on host galaxies on the
+        catsim database.
+    - FrozenSNCat: catalogs that are 'frozen' on the catsim database. For a
+        user to use one of these catalogs, such a catalog would have to be
+        uploaded to catsim. Examples of such catalogs that are on the catsim
+        database are the tables `TwinkSN` and `TwinkSNKraken`
 """
 import numpy as np
 
 from lsst.sims.catalogs.definitions import InstanceCatalog
 from lsst.sims.catalogs.decorators import compound
-from lsst.sims.photUtils import BandpassDict
+from lsst.sims.photUtils import (BandpassDict, Bandpass)
 from lsst.sims.catUtils.mixins import CosmologyMixin
 from lsst.sims.catUtils.mixins import PhotometryBase
```

Comment:
It looks like you don't use `PhotometryBase`.  You should remove the import statement.